### ISSUE TITLE
New ownCloud iOS app uploading images as FullSizeRender if EXIF data was removed from image

### ISSUE THREAD

**Issue by alexhk** [owncloud/ios-app#520]
The old iOS app could upload images with removed EXIF info without problems.

The new iOS app uploads these images as "FullSizeRender.jpg".

Uploaded using old iOS app (via owncloud Share-In extension within the iOS photo gallery):

```
Photo-2019-10-06-16-39-57_1869.JPG  2.5 MB
Photo-2019-10-06-16-32-43_1868.JPG  6.2 MB
```

Uploaded using new iOS app (via app itself, as the iOS photo gallery doesn't show a share to the new app):

```
IMG_1869.jpg       2.5 MB
FullSizeRender.jpg 6.2 MB
```

The new one however uploads images 

Steps to reproduce
1. Start iOS app.
2. Settings "Convert HEIC to JPEG" - ON (without this it won't upload anything, without error message)
3. Hit +, Upload from your photo library, select 2 photos, one original and one where the EXIF data has been altered (ie. GPS info removed or EXIF completely removed).
4. Check what is being uploaded.

### Expected behaviour

iOS app should upload all images using original file name, not "FullSizeRender.jpg" for the one where EXIF has been altered.

### Actual behaviour

iOS app uploads with file name "FullSizeRender.jpg".

### Client
**iOS version:** 13.1.2

**ownCloud app version:** 1.1.0 (142)

**Device model:** iPhone 8

### Server configuration
**Web server:** Ubuntu 18.04

**Database:** MySQL

**PHP version:** 7.2

**ownCloud version:** 10.2

**Storage backend (external storage):**

**Comment by hosy** [comment]
@alexhk this is fixed in version 1.1.1. Thank you for opening this issue!

**Comment by jesmrec** [comment]
Done and released
